core/runtime/src/main/java/io/quarkus/runtime/configuration/AbstractDelegatingConfigSource.java

```diff
@@ -31,7 +31,7 @@ public AbstractDelegatingConfigSource(final ConfigSource delegate) {
      *
      * @return the delegate config source (not {@code null})
      */
-    protected @NotNull ConfigSource getDelegate() {
+    public @NotNull ConfigSource getDelegate() {
```

Comment:
I need to access to the delegate to check the test `ConfigSource` which contains setters method to add values programmatically. You'll see it later.